python/spatialdata-js-util/src/spatialdata_js_util/codecs/encoding.py

```
@@ -15,21 +15,46 @@
+    info = np.iinfo(dtype)
+    return 1.0 / (int(info.max) - int(info.min) + 1)
+
+
+def htj2k_encode_options(
+    encode_options: dict[str, Any], *, dtype: np.dtype | None = None
+) -> tuple[bool, float]:
     """Map encode options to the OpenJPH ``(reversible, quality)`` pair.
 
     ``quality`` is the OpenJPH quantization step: lower means higher fidelity and
-    a larger codestream. It is *not* a JPEG-style 0–100 quality.
+    a larger codestream. It is *not* a JPEG-style 0–100 quality. It is relative
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)